### PR TITLE
[Feature] Make plugins installable

### PR DESCRIPTION
**Pull request by gondar00**
Fixes: #3870

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2h1anyv Make Plugins Installable](https://app.clickup.com/t/2h1anyv)

File: server/src/helpers/plugins.helper.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  async getService(pluginId: string, kind: string) {
+    try {
+      if (pluginId) {
+        let decoded: string;
+        if (this.plugins[pluginId]) {
+          decoded = this.plugins[pluginId];
+        } else {
+          const plugin = await this.pluginsRepository.findOne({ where: { id: pluginId } });
+          decoded = decode(plugin.indexFile.data.toString());
```
Comment (akshaysasidrn) [inline server/src/helpers/plugins.helper.ts:34]: Shouldn't we enforce utf-8 decode here as well?

Comment (gondar00) [reply server/src/helpers/plugins.helper.ts:34]: default encoding is `utf-8` for `toString`

**Review by akshaysasidrn — DISMISSED** [review]

**Comment by gondar00** [comment]
@SocketSecurity ignore esbuild@0.15.8

**Comment by gondar00** [comment]
@akshaysasidrn please ignore the [Socket Security](https://socket.dev/) error for time being

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**